pmm-app/tests/verifyMysqlDashboards_test.js

```diff
@@ -97,6 +97,34 @@ Scenario(
   },
 );
 
+Scenario(
+  'PMM-T348 - PXC/Galera Node Summary dashboard @not-pr-pipeline',
```

Comment:
The nightly job runs all dashboard related tests, please add `@not-ui-pipeline @nightly` to both